src/accounts/DeleteFreeAccountOverlay.tsx

```diff
@@ -51,11 +52,11 @@ const DeleteOrgOverlay: FC = () => {
       tier: account.type,
       email: user.email,
     }
-    event('DeleteOrgDismissed Event', payload)
+    event('DeleteAccountDismissed Event', payload)
 
     if (isFlagEnabled('rudderstackReporting')) {
       // Send to Rudderstack
-      track('DeleteOrgDismissed', payload)
+      track('DeleteAccountDismissed', payload)
```

Comment:
same comment here about `track` not being necessary. We're sending two events to our analytics tooling, one called `DeleteAccountDismissed Event` and one called `DeleteAccountDismissed`

Reply:
@hoorayimhelping On the event name changes - Beth has validated the event name changes with analytics

Reply:
@wdoconnell meaning that the ones in the PR are already what they expect?

is that relevant to this comment about `track` or just in general?

Reply:
@hoorayimhelping  yes! i've sent the list to mona and she has updated the change on her end